trl/trainer/utils.py

```diff
@@ -641,9 +641,9 @@ def peft_module_casting_to_bf16(model):
     for name, module in model.named_modules():
         if isinstance(module, BaseTunerLayer):
             module = module.to(torch.bfloat16)
-        if "norm" in name:
+        elif isinstance(module, torch.nn.LayerNorm) or "norm" in name:
```

Comment:
@pacman100 is it correct to assume that you want to cast here all layernorms?

Reply:
Hello Younes, yes.